Symfony/CS/Fixer.php

```diff
@@ -139,18 +140,26 @@ public function fix(ConfigInterface $config, $dryRun = false, $diff = false)
             $config->getRules()
         );
 
-        foreach ($config->getFinder() as $file) {
-            if ($file->isDir() || $file->isLink()) {
-                continue;
-            }
+        $iterator = $config->getFinder();
+        if ($iterator instanceof \IteratorAggregate) {
+            $iterator = $iterator->getIterator();
+        }
+        /* @var \Iterator $iterator */
```

Comment:
It was fixed by `PhpdocToCommentFixer`. ;)
I will move this line to variable declaration.